edit.js

```diff
@@ -1115,12 +1094,12 @@ const _bindWorld = w => {
     _pushWorld(hash);
   });
 };
-(async () => {
```

Comment:
I'm pretty sure the code universally uses spaces between modifiers and function tokens, so the prior version is preferred.

https://eslint.org/docs/rules/space-before-function-paren

Reply:
I've changed it to:

```
        "space-before-function-paren": ["error", {
            "anonymous": "never",
            "named": "never",
            "asyncArrow": "always"
        }]
```

so that we still don't use a space after 'normal' function names and their brackets like `function name(param) { ... }` (which I think is what the code currently does)